include/RMGMasterGenerator.hh

```diff
@@ -29,21 +29,27 @@ class RMGMasterGenerator : public G4VUserPrimaryGeneratorAction {
 
   public:
 
+    /**
+     * @brief Enumeration for specifying the primary vertex confinement strategy.
+     */
     enum class Confinement {
-      kUnConfined,
-      kVolume,
-      kFromFile,
+      kUnConfined, ///< No confinement is applied; the primary vertex is generated freely.
```

Comment:
"generated freely" does not make sense (there is no "generation" involved _at all_). It follows the settings used in the primary generator (i.e. GPS)